pkg/networkmanager/firewall.jsx

```diff
@@ -777,7 +800,39 @@ export class Firewall extends React.Component {
     }
 
     onRemoveZone(zone) {
-        firewall.deactiveateZone(zone);
+        let body;
+        if (firewall.zones[zone].services.indexOf("cockpit") !== -1)
+            body = <Alert type="warning">{_("This zone contains the cockpit service. Removing the zone risks making the web console unreachable.")}</Alert>;
+        this.setState({
+            deleteConfirmationModal: <DeleteConfirmationModal title={ cockpit.format(_("Remove zone $0"), zone) }
+            body={body}
```

Comment:
I feel it needs some more context in the body to explain why this is a destructive action. What about:
_Removing the zone will remove all services within it._